main.go

```diff
@@ -145,10 +149,15 @@ func mainerr() error {
 			}
 			gopath = filepath.Join(uhd, "go")
 		}
-
 		// TODO I don't think the module cache path is advertised anywhere public...
 		// intentionally but in case it is, replace what follows
-		localCacheProxy = "GOPROXY=file://" + path.Join(filepath.ToSlash(gopath), "pkg", "mod", "cache", "download")
+		cachePath := path.Join(filepath.ToSlash(gopath), "pkg", "mod", "cache", "download")
+		if goenv.ReleaseTags["go1.13"] && cachePath[0] != '/' {
+			// in Go 1.13 the handling of file:// proxy URLs changed to require a /
```

Comment:
This comment seems odd, because I'm sure that at least *one* `/` was required before... is it that Windows absolute paths require three slashes now?

Reply:
> is it that Windows absolute paths require three slashes now?

Correct. 

Because we are deriving the value of `gopath` here from `go env GOPATH` we are assured it is a valid value (i.e. absolute), because otherwise `go env` complains and we bail before this point. This is true for all versions under consideration `>= 1.11`

If we have an absolute path I think this also implies we cannot have a network location.

Which means that the above check is then correct. We could tighten it slightly to only apply on Windows, but in reality that's the only place that such a condition would hold true in any case.